bin/solr-jsonl-chunk-loader.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -e
+
+scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $scriptDir/common_routines.sh
+
+require_env_var INPUT_JSONL
+require_env_var SOLR_COLLECTION
+require_env_var SCHEMA_VERSION
+
+COLLECTION=${SOLR_COLLECTION}-v${SCHEMA_VERSION}
+HOST=${SOLR_HOST:-'localhost:8983'}
+# SOLR_PROCESSORS must be null or a comma-separated list of processors to use during an update
```

Comment:
can it be indicated which values can `$SOLR_PROCESSORS` take? It looks like an optional parameters, sometimes defined as `dedup`

Reply:
@alfonsomunozpomer would be the man here to know what processors to use... however, I wouldn't delay merging this due to this.

Reply:
@pmb59 It really depends on the collection/schema. Since `solr-jsonl-chunk-loader.sh` is a general-purpose script, it’s up to the caller to set `SOLR_PROCESSORS`.